src/components/Account/Account.tsx

```diff
@@ -232,7 +237,9 @@ const Account = ({ panelClassName, panelHeaderClassName, canUpdateEmail = true }
         formSection({
           label: t('account.terms_and_tracking'),
           saveButton: t('account.update_consents'),
-          onSubmit: (values) => updateConsents(formatConsentsFromValues(publisherConsents, values)),
+          onSubmit: async (values) => {
+            await updateConsents(formatConsentsFromValues(publisherConsents, values));
+          },
```

Comment:
Another merge / branch artifact:
```suggestion
          onSubmit: (values) => updateConsents(formatConsentsFromValues(publisherConsents, values)),
```

Reply:
yes, but fails on lint :/

Reply:
done